addons/repair/XEH_postInit.sqf

```diff
@@ -18,3 +18,62 @@
         _object allowDamage false;
     };
 }] call CBA_fnc_addEventHandler;
+
+
+// placed in editor static objects don't trigger init
+{
+    if (local _x && {getRepairCargo _x > 0}) then {
+        _x setRepairCargo 0;
+    };
+} forEach allMissionObjects "Static";
+
+["All", "InitPost", {
+    params ["_vehicle"];
+    if (local _vehicle && {getRepairCargo _vehicle > 0}) then {
+        _vehicle setRepairCargo 0;
+    };
+}, true, ["Man"], true] call CBA_fnc_addClassEventHandler;
+
+["CAManBase", "InitPost", {
+    params ["_unit"];
+    if (local _unit && {_unit getUnitTrait "engineer"}) then {
+        _unit setUnitTrait ["engineer", false];
+        if (-1 isEqualTo (_unit getVariable ["ACE_IsEngineer", -1])) then {
```

Comment:
This is better:
```suggestion
        if !(_unit getVariable ["ACE_IsEngineer", false]) then {
```

Reply:
No, unit variable can be set to false while maintaining engineer trait. -1 is more appropriate as it's only valid if the variable hasn't been set.

Reply:
In that sense yeah, since would broadcast for nothing.